staging/src/k8s.io/node-api/pkg/apis/node/v1alpha1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +k8s:openapi-gen=true
```

Comment:
NOTE TO REVIEWERS:

Should this be here? It looks like it causes an entry in `k8s.io/kubernetes/pkg/generated/openapi` to be generated, but I think this is supposed to be completely out of tree?